cellfinder_core/classify/classify.py

```diff
@@ -74,3 +87,18 @@ def main(
         points_list.append(cell)
 
     return points_list
+
+
+def noop(*args, **kwargs):
+    pass
```

Comment:
can you add a docstring to explain the purpose of this?

Reply:
Thanks for catching - I don't think this is actually needed, so I've removed it and modified the new class accordingly.

Reply:
I'd never come across `NOOP` before, was a bit confused